mibs/ttc/TTC-NOTIFICATIONS-MIB.mib

```diff
@@ -0,0 +1,945 @@
+TTC-NOTIFICATIONS-MIB DEFINITIONS ::= BEGIN
```

Comment:
What is this MIB?  It has references to icinga in it?

(also the file name is wrong, it should match the mib name.)

Reply:
Is a MIB file created for Icinga & LibreNMS notifications. Net-snmp lookup seems to work fine with that file name. Nonetheless, will rename from TTC-NOTIFICATIONS-MIB.mib to TTC-NOTIFICATIONS-MIB as requested.